test_patches_locally.sh

```diff
@@ -4,12 +4,19 @@ FILENAME="ray-${VERSION}.tar.gz"
 URL="https://github.com/ray-project/ray/archive/${FILENAME}"
 DIRNAME="ray-ray-${VERSION}"
 
+CACHED="$TMPDIR/$FILENAME"
+
+if [ ! -f $CACHED ]; then
+  echo "Downloading $FILENAME to $CACHED"
+  curl -Lqo "$CACHED" "$URL"
+fi
+
```

Comment:
This is just for local testing to avoid re-downloading the same archive over and over again